prompt/deps.ts

```diff
@@ -8,9 +8,10 @@ export {
   stripColor,
   underline,
   yellow,
-} from "https://deno.land/std@0.196.0/fmt/colors.ts";
+} from "https://deno.land/std@0.214.0/fmt/colors.ts";
 export {
   dirname,
   join,
   normalize,
-} from "https://deno.land/std@0.196.0/path/mod.ts";
+} from "https://deno.land/std@0.214.0/path/mod.ts";
+export type { Reader } from "https://deno.land/std@0.214.0/io/types.ts";
```

Comment:
I would prefere to upgrade dependencies in a seperate PR. Also, the exported type `Reader` seems to be unused.